ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master.yaml

```diff
@@ -24,7 +24,6 @@ tag_specification:
   namespace: ocp
 tests:
 - as: gpu-operator-e2e
-  cron: 0 */23 * * *
```

Comment:
So test every time we PR against the repo? this tests take like 1 hour, and use a GPU, I prefer to have them just as Nightly

Reply:
not automatically at every, `always_run: false` only when hitting `/test gpu-operator-e2e` on the PR